### DISCUSSION TITLE
Requesting example input and output data for a step-by-step follow-up

### DISCUSSION THREAD

**Discussion by Haris888** [btschwertfeger/BiasAdjustCXX#42] [Q&A]
Hello, I am wandering if is it possible to provide examplne input and output data for a step-by-step follow up? 
Thanks a lot.
-Haris

**Comment by btschwertfeger** [comment]
Hello @Haris888, you can find some input data in the `input_data` directory of this repository. It provides modeled data for the control and scenario period as well as reference ("observational") data for the control period.  

Since I have no rights on other data than that, I can just refer to: 
- https://esgf-data.dkrz.de/search/cmip6-dkrz/
- https://esgf-data.dkrz.de/search/cordex-dkrz/
- https://psl.noaa.gov/data/gridded/data.20thC_ReanV3.html#caveat

A hands-on example for applying the BiasAdjustCXX tool on the example data provided in this repository can be found here: https://github.com/btschwertfeger/BiasAdjustCXX/blob/v1.9.1/examples/Hands-On-BiasAdjustCXX.ipynb.

In general I recommend reading the official [documentation](https://biasadjustcxx.readthedocs.io/en/latest/). It is important to understand the different methods, before applying those to the data. The requirements of the input data are described there as well (e.g. same spatial resolution (for the delta method you also need to have the same temporal dimension), "lat", "lon", "time" as dimensions).  

If you are familiar with Python, you may also want to check out [python-cmethods](https://github.com/btschwertfeger/python-cmethods) it servers a lot of additional [examples](https://github.com/btschwertfeger/python-cmethods/blob/v1.0.3/examples/examples.ipynb) + [documentation](https://python-cmethods.readthedocs.io/en/latest/) (you may want to wait with that one more week, since I'm preparing the v2.0.0 release for that package).
